File: fe_flutter_ui/lib/models/news.dart
class News {
  int? id;
  String? userId;
  String? title;
  String? newsDate;
  String? image;
  String? imageFile;
  String? description;
  bool? status;

  News({
    this.id,
    this.userId,
    this.title,
    this.newsDate,
    this.image,
    this.imageFile,
    this.description,
    this.status,
  });

  News.fromJson(Map<String, dynamic> json) {
    id = json['id'];
    userId = json['userId'];
    title = json['title'];
    newsDate = json['newsDate'];
    image = json['image'];
    imageFile = json['imageFile'];
    description = json['description'];
    status = json['status'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = <String, dynamic>{};
    data['id'] = id;
    data['userId'] = userId;
    data['title'] = title;
    data['newsDate'] = newsDate;
    data['image'] = image;
    data['imageFile'] = imageFile;
    data['description'] = description;
    data['status'] = status;

    return data;
  }
}

File: be_aspnet_api/MrSoaiAPI/Model/News.cs
using System.ComponentModel;
using System.ComponentModel.DataAnnotations.Schema;

namespace MrSoaiAPI.Model
{
    public class News
    {
        public int Id { get; set; }
    
        public string UserId { get; set; }
    
        public string Title { get; set; }
    
        public DateTime NewsDate { get; set; }
        public string Image { get; set; }
        [NotMapped]
        public IFormFile ImageFile { get; set; }
        public string Description { get; set; }
        [DefaultValue(true)]
        public bool Status { get; set; } = true;
        public  MrSoaiUser User { get; set; }
    }
}
